lib/rucio/common/.github/workflows/build.yaml

```diff
@@ -0,0 +1,43 @@
+name: Build Rucio Hermes Kafka container
```

Comment:
This file isn't used either, is it?

Reply:
yes it is;  It is used to build the container that Brandon deploys.